### ISSUE TITLE
Redstone dust is not picked up correctly when sieving

### ISSUE THREAD

**Issue by jpmac26** [MikeLydeamore/ExNihiloAdscensio#93]
This is a fairly minor issue, but almost every item that you get from sieving is propelled directly to the player and immediately picked up within about half a second, noticeably faster than the time it takes to pick up an item that is dropped on the ground by a player, which takes about 1-1.5 seconds. However, Redstone dust does not behave like this, and instead takes the usual 1-1.5 seconds to be picked up. Furthermore, because it is still propelled towards the player like the other items, the Redstone dust always ends up flying around 5-10 blocks behind the player. This happens on both singleplayer and multiplayer, and can be particularly frustrating when playing a modpack like SkyFactory 3, where it's easy to lose 90% of your Redstone because it goes flying off the edge of the world.

TL;DR: I think this is a bug because no other items have a pickup delay when sieving, so perhaps the way that the Redstone dust item is being placed in the world is incorrect? Hopefully it's an easy fix :)

**Comment by MikeLydeamore** [comment]
Emphasis on hopefully.

**Comment by jpmac26** [comment]
Lemme build the new code, and I'll let you know for sure ;)
